opcua.cpp

```diff
@@ -842,7 +1009,7 @@ int OPCUA::subscribe()
 			done = true;
 		}
 
-		logger->debug("MI_AddMonitoredItems_Call: %d %d %d", i, numNodeIds, totalMonitoredItems);
+		logger->info("MI_AddMonitoredItems_Call: i=%d, numNodeIds=%d, totalMonitoredItems=%d, m_nodes.size()=%d", i, numNodeIds, totalMonitoredItems, m_nodes.size());
```

Comment:
I disagree with the change. I think this should remain at _debug_ log level. Why are you changing it?

Reply:
Same reason as previous comment. But I'll just revert this. I needed this log to debug the issue in FOGL-8283.